shell/platform/android/io/flutter/embedding/engine/FlutterEngineGroup.java

```diff
@@ -155,7 +159,16 @@ public FlutterEngine createAndRunEngine(@NonNull Options options) {
       engine.getDartExecutor().executeDartEntrypoint(dartEntrypoint, dartEntrypointArgs);
     } else {
       engine =
-          activeEngines.get(0).spawn(context, dartEntrypoint, initialRoute, dartEntrypointArgs);
+          activeEngines
+              .get(0)
+              .spawn(
+                  context,
+                  dartEntrypoint,
+                  initialRoute,
+                  dartEntrypointArgs,
+                  platformViewsController,
+                  automaticallyRegisterPlugins,
+                  waitForRestorationData);
```

Comment:
When creating the first engine (spawner), we also need to pass new parameters to it.

Reply:
Good advice !  I'll fill it in later